test/powershell/Modules/Microsoft.PowerShell.Management/New-Item.Tests.ps1

```diff
@@ -386,3 +386,13 @@ Describe "New-Item -Force allows to create an item even if the directories in th
         $FullyQualifiedFile | Should -Exist
     }
 }
+
+Describe "New-Item -Force should throw an error for invalid characters in directory path" -Tags "CI" {
+    BeforeAll {
+        $invalidPath = Join-Path -Path $TestDrive -ChildPath 'Invalid?'
```

Comment:
The test should work on Linux and MacOS too.

Reply:
As mentioned before, I was not able to force invalid directory name error on Ubuntu, therefore I just propose to skip that test when running on Linux or macOS.

Reply:
I think it's fine to limit the tests to Windows. The only way to construct an invalid directory path that I'm aware of on Unix-like platforms is to include `NUL` (`` `0 ``) characters, but there's a dedicated check for that already in place (`Null character in path.`)